pyscf/gto/basis/soecp/ECPDS78MDFSO.dat

```diff
@@ -0,0 +1,127 @@
+ECP
+# CHECKME 78? 60?
+# Q=22., MEFIT, MCDHF+Breit+QED, Ref 29.
+# [29] B. Metz, H. Stoll, M. Dolg, J. Chem. Phys. 113, 2563 (2000).
+Pb nelec 60
+Pb ul
+2	1.000000	0.000000
+Pb S
+2	1.940106	35.775442
+2	0.275063	-0.558568
+Pb P
+2	0.921930	2.609796	-2.144958
+2	0.903965	5.553646	-2.666379
+Pb D
+2	0.664793	2.956364	-5.219593
+2	0.621886	4.242489	5.553646
+Pb F
+2	0.813760	-2.144958	-2.956364
```

Comment:
SO coefficient is wrong here, see last line in 
www.tc.uni-koeln.de/cgi-bin/pp.pl?language=en,format=molpro,element=Pb,job=getecp,ecp=ECP78MDF